packages/graphql-transformer-common/src/ResourceConstants.ts

```diff
@@ -51,6 +51,10 @@ export class ResourceConstants {
 
         // Auth
         AuthCognitoUserPoolId: 'AuthCognitoUserPoolId',
+        AuthCognitoUserPoolName: 'AuthCognitoUserPoolName',
```

Comment:
I believe these have been removed from master. Can you remove them as well?

Reply:
done - removed the unused constants.